language_service/src/signature_help/tests.rs

```diff
@@ -0,0 +1,623 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use super::get_signature_help;
+use crate::test_utils::{compile_with_fake_stdlib, get_source_and_marker_offsets};
+use expect_test::{expect, Expect};
+use indoc::indoc;
+
+/// Asserts that the hover text at the given cursor position matches the expected hover text.
```

Comment:
Comment seems copy-pasted from hover, update the wording?

Reply:
Oh good catch! Fixed